google-cloud-clients/google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1beta1/stub/EnhancedBigQueryStorageStubSettingsTest.java

```diff
@@ -120,7 +124,7 @@ public void testCreateReadSessionSettings() {
   public void testReadRowsSettings() {
     ServerStreamingCallSettings.Builder<ReadRowsRequest, ReadRowsResponse> builder =
         EnhancedBigQueryStorageStubSettings.newBuilder().readRowsSettings();
-    assertThat(builder.getRetryableCodes()).containsAllOf(Code.DEADLINE_EXCEEDED, Code.UNAVAILABLE);
+    assertThat(builder.getRetryableCodes().containsAll(retryCodeList));
```

Comment:
Please update this as well to `containsAtLeast` or with `isTrue()`

Reply:
done